Doc/library/sys.rst

```diff
@@ -1235,6 +1235,13 @@ always available.
 
    .. versionadded:: 3.13
 
+.. function:: _jit_enabled()
+   Return :const:`True` if :ref:`JIT compiler <whatsnew313-jit-compiler>` is active.
+   Return :const:`False` otherwise.
+
+   Available only in JIT builds, tier 2 interpreter doesn't count here.
+
+   .. versionadded:: next
```

Comment:
```suggestion

.. function:: _jit_enabled()

   Return :const:`True` if the :ref:`JIT compiler <whatsnew313-jit-compiler>` is active,
   and :const:`False` otherwise.

   Available only in JIT builds; tier 2 interpreter does not count here.

   .. versionadded:: next

```